homeassistant/components/teslemetry/icons.json

```diff
@@ -84,6 +84,28 @@
           "off": "mdi:hvac-off",
           "on": "mdi:hvac"
         }
+      },
+      "backup_capable": {
+        "state": {
+          "off": "mdi:battery-off",
+          "on": "mdi:home-battery"
+        }
+      },
+      "grid_status": {
+        "state": {
+          "off": "mdi:transmission-tower-off",
+          "on": "mdi:transmission-tower"
+        }
+      },
+      "grid_services_active": {
+        "state": {
+          "on": "mdi:sine-wave"
+        }
+      },
+      "components_grid_services_enabled": {
+        "state": {
+          "on": "mdi:sine-wave"
+        }
```

Comment:
should we set a default?

Reply:
I couldn't find a default so wanted to use the regular binary sensor default. Do I need to find and set that explicitly?

Reply:
Ok i've added off icons for those two. I didnt think a default was a good idea since none of the custom icons are meaningful when the state is unknown or unavaliable.